pydantic/_internal/_model_construction.py

```diff
@@ -34,7 +33,7 @@
     _make_forward_ref,
     eval_type_backport,
     is_annotated,
-    is_classvar,
+    is_classvar_annotation,
```

Comment:
Should we follow the same pattern here and just do `import _typing_extra` then call associated methods like `_typing_extra.XXX`?

Reply:
We would have to change all the `from ._x import ...` in this file to be consistent, so I left it as is